src/test/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilderTest.java

```diff
@@ -101,7 +101,8 @@ void generateFilterForDomainCriteriaThrowsIfGivenUnknownDomain() {
   @ParameterizedTest
   @ArgumentsSource(QueryTestUtils.Contexts.class)
   void generateFilterIdentifiesDomainCriteria(SqlRenderContext context) {
-    SnapshotBuilderCriteria criteria = generateDomainCriteria();
+    SnapshotBuilderCriteria criteria =
+        generateDomainCriteria(SnapshotBuilderTestData.CONDITION_OCCURRENCE_DOMAIN_ID, 0);
```

Comment:
Why add the concept id 0 here?

Reply:
I was hoping to surface that we're setting the concept id to 0 instead of hiding in the method. But, it is registering a code smell, so I've removed this parameter.